loaders/src/glTF/babylon.glTFFileLoaderInterfaces.ts

```diff
@@ -102,16 +108,33 @@ module BABYLON {
         name?: string;
     }
 
-    export interface IGLTFAccessor extends IGLTFChildRootProperty {
+    export interface IGLTFAccessorSparseIndices {
         bufferView: number;
-        byteOffset: number;
-        byteStride?: number;
+        byteOffset?: number;
+        componentType: EIndicesComponentType;

```

Comment:
I would just use EComponentType here. JSON parsing doesn't validate anyways, so creating a separate enum doesn't do anything for this.